sufia-models/app/services/sufia/create_derivatives_service.rb

```diff
@@ -0,0 +1,17 @@
+module Sufia
+  class CreateDerivativesService
+    include Hydra::Derivatives
+
+    def self.run(generic_file)
+      new(generic_file).create_derivatives
```

Comment:
That's correct. But the `new` creats an instance of the CreateDerivativesService.